doc/tutorials/introduction.md

```diff
@@ -24,7 +24,7 @@ Although Closure library functions are widely used within OL3, none of them are
 Unlike in, say, Node, where a module's exports are fixed in the source, with Closure Compiler, exports can be defined at compile time. This makes it easy to create builds that are customized to the needs of a particular site or application: a __custom build__ only exports those properties and methods needed by the site or application. As the full build is large, and will probably become larger as new features are added to the API, it's recommended that sites create a custom build for production software.
 
 ## Renderers and Browser Support
-The library currently includes three renderers: Canvas, DOM, and WebGL. All three support raster data from tile/image servers, but only the Canvas renderer currently supports vector data. This means that only those browsers that [support Canvas](http://caniuse.com/canvas) can handle vector data. In particular, this excludes Internet Explorer versions before 9, though there is some support for those in the DOM renderer. Clearly, the WebGL renderer can only be used on those devices and browsers supporting WebGL.
+The library currently includes three renderers: Canvas, DOM, and WebGL. All three support both raster data from tile/image servers, and vector data; WebGL however only supports Point vectors. Clearly only those browsers that [support Canvas](http://caniuse.com/canvas) can use the Canvas renderer; this excludes Internet Explorer versions before 9. Equally, the WebGL renderer can only be used on those devices and browsers supporting WebGL.
```

Comment:
Maybe in this context we could also mention that the WebGL renderer does not support labels?